package-lock.json

```diff
@@ -1193,7 +1193,7 @@
     "is-buffer": {
```

Comment:
This file?

Reply:
It's to ensure dependencies aren't bumped up impliciltly.  https://docs.npmjs.com/files/package-locks